xbmc/interfaces/legacy/DrmCryptoSession.h

```diff
@@ -143,12 +162,15 @@ namespace XBMCAddon
       /// @brief \python_func{ ProvideKeyResponse(response) }
       ///-----------------------------------------------------------------------
       /// \anchor xbmcdrm_ProvideKeyResponse
-      /// Provide key data returned from key server. See getKeyRequest(...)
+      /// When a key response is received from the license server,
+      /// must be sent to the %DRM instance by using provideKeyResponse.
+      /// See also \ref xbmcdrm_GetKeyRequest "GetKeyRequest".
       ///
-      /// @param      [byte] response Key data returned from key server
+      /// @param      response   byte - Key data returned from the license server
       ///
-      /// @return     String If the response is for an offline key requests a keySetId which can be used later
-      ///                    with restoreKeys, else return empty for streaming key requests.
+      /// @return     A keySetId if the response is for an offline key requests which
+      ///             can be used later with restoreKeys,
+      ///             else return empty for streaming key requests.
       ///
       ///------------------------------------------------------------------------
       /// @python_v18 New function added.
```

Comment:
`@python_v19 With python 3 the response argument must be a bytearray instead of byte.`

Reply:
done